account_move_transactionid_import/parser/transactionid_file_parser.py

```diff
@@ -36,20 +36,19 @@ def __init__(
             "commission_amount": float_or_zero,
         }
         super().__init__(
-            profile,
+            journal,
             extra_fields=conversion_dict,
             ftype=ftype,
             header=header,
             **kwargs
         )
-        self.support_multi_moves = True
 
     @classmethod
     def parser_for(cls, parser_name):
         """Used by the new_bank_statement_parser class factory. Return true if
         the providen name is generic_csvxls_transaction
```

Comment:
hook to check parser .....

Reply:
I did removed it altogether since the classes it's inheriting from already return False ;)